modular_skyrat/modules/gunsgalore/code/guns/akm.dm

```diff
@@ -33,16 +33,83 @@
+	icon_state = "akm_xeno"
+	ammo_type = /obj/item/ammo_casing/realistic/a762x39/xeno
+
+/obj/item/ammo_box/magazine/akm/ap
+	name = "krinkov magazine (ARMOR PIERCING)"
+	desc = "a banana-shaped double-stack magazine able to hold 30 rounds of 5.6mm ammo. It's said that in the early days of SolFed's spread, Spanish colony rebels often referred to these as 'Goat Horns'. Contains armor-piercing ammunition."
+	icon_state = "akm_ap"
+	ammo_type = /obj/item/ammo_casing/realistic/a762x39/ap
+
+/obj/item/ammo_box/magazine/akm/emp
+	name = "krinkov magazine (EMP)"
+	desc = "a banana-shaped double-stack magazine able to hold 30 rounds of 5.6mm ammo. It's said that in the early days of SolFed's spread, Spanish colony rebels often referred to these as 'Goat Horns'. Contains ion ammunition, good for disrupting electronics and wrecking mechas."
+	icon_state = "akm_emp"
+	ammo_type = /obj/item/ammo_casing/realistic/a762x39/emp
+
+/obj/item/ammo_box/magazine/akm/rubber
+	name = "krinkov magazine (RUBBER)"
+	desc = "a banana-shaped double-stack magazine able to hold 30 rounds of 5.6mm ammo. It's said that in the early days of SolFed's spread, Spanish colony rebels often referred to these as 'Goat Horns'. Contains less-than-lethal rubber ammunition."
+	icon_state = "akm_rubber"
+	ammo_type = /obj/item/ammo_casing/realistic/a762x39/civilian/rubber
+
 /obj/item/ammo_box/magazine/akm/banana
 	name = "\improper Krinkov extended magazine"
 	desc = "a banana-shaped double-stack magazine able to hold 45 rounds of 5.6x40mm ammunition. It's meant to be used on a light machine gun, but it's just a longer Krinkov magazine."
 	max_ammo = 45
 
+/obj/item/ammo_box/magazine/akm/civvie
+	name = "\improper Sabel magazine"
+	desc = "a shortened double-stack magazine able to hold 15 rounds of civilian-grade 5.6mm ammo."
+	icon_state = "akm_civ"
+	max_ammo = 15
+	ammo_type = /obj/item/ammo_casing/realistic/a762x39/civilian
+
 /obj/item/gun/ballistic/automatic/akm/modern
 	name = "\improper Bubba's Krinkov"
 	desc = "A modified version of the most iconic human firearm ever made. Most of the original parts are gone in favor of aftermarket replacements."
 	icon_state = "akm_modern"
 	inhand_icon_state = "akm"
 	worn_icon_state = "akm"
 	fire_delay = 1
+
+/obj/item/gun/ballistic/automatic/akm/civvie
+	name = "\improper Sabel-42 carbine"
+	desc = "A timeless human design of a carbine chambered in the NRI's 5.6mm ammo. The internal modifications made to the firearm in order to accommodate for non-military use made it incompatible with conventional munitions and gave it the inability to fire fully automatic. It's purpose-built to fire low-grade civilian ammo, anything stronger would obliterate the rifling and render the firearm useless."
+	icon_state = "akm_civ"
+	inhand_icon_state = "akm_civ"
+	fire_select_modes = list(SELECT_SEMI_AUTOMATIC)
```

Comment:
i'd say either give it full auto, (with that long-ass fire delay), or trim off the recoil/spread, as it stands it's going to be far-far behind even the mosin/likeanyhandgunhere stat-wise (and probably slip out of that 5-6s time-to-down against unarmored we aim for)

Reply:
> i'd say either give it full auto, (with that long-ass fire delay), or trim off the recoil/spread, as it stands it's going to be far-far behind even the mosin/likeanyhandgunhere stat-wise (and probably slip out of that 5-6s time-to-down against unarmored we aim for)

Full auto is *completely* contradictory with the gun's inspiration, so, yes, I'll remove some of the recoil.